project1.html

```diff
@@ -1,121 +1,205 @@
+              </li>
+              <li>
+                <a><img src="Linkedinicon.jpg" alt=" " class="linkedin" /></a>
+              </li>
+              <li>
+                <a><img src="angellist.jpg" alt=" " class="angellist" /></a>
+              </li>
+              <li>
+                <a><img src="twitter.jpg" alt=" " class="twitter" /></a>
+              </li>
+              <li>
+                <a><img src="medium.jpg" alt=" " class="medium" /></a>
+              </li>
+            </ul>
+          </div>
+          <div><button type="button" class="two">Get my resume</button></div>
+        </div>
+
+        <div class="flex-container2">
+          <div><h4 class="languages">Languages</h4></div>
+          <div class="flex-container">
+            <div class="flex-container3">
+              <div>
+                <a><img src="Ellipse1.jpg" alt="Javascript" /></a>
+              </div>
+              <div><h5>Javascript</h5></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+            </div>
+            <div class="flex-container4">
+              <div>
+                <a><img src="Ellipse1(1).jpg" alt="HTML" /></a>
+              </div>
+              <div><h5>HTML</h5></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+            </div>
+            <div class="flex-container5">
+              <div>
+                <a><img src="Ellipse1(2).jpg" alt="CSS" /></a>
+              </div>
+              <div><h5>CSS</h5></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+            </div>
+          </div>
+          <div>
+            <a class="Vector1"><img src="Vector1.png" alt="" /></a>
+            <a class="Vector2"><img src="Vector2.png" alt="" /></a>
+            <a class="Vector3"><img src="Vector2.png" alt="" /></a>
+            <div><h4 class="Frameworks">Frameworks</h4></div>
+            <a class="Separator1"><img src="SeparatorBottom.jpg" alt="" /></a>
+          </div>
+          <div><h4 class="Skills">Skills</h4></div>
+          <a class="Separator2"><img src="SeparatorBottom.jpg" alt="" /></a>
+          <a class="Separator1"><img src="SeparatorBottom.jpg" alt="" /></a>
+        </div>
+
+        <div class="add-contact add-contact-bckgrnd">
+          <div class="image"></div>
+          >
+
+          <h1>Contact me</h1>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+
+          <form
+            action="https://formspree.io/f/xzboleol"
+            method="POST"
+            class="form"
+          >
+            <ul>
```

Comment:
- [ ] `required`
- [ ] Kindly fix the form properly like the previous reviewer your submission button is not working. please see this [form spree configuration](https://formspree.io/html/)